lib/cgo/cipher.crypto.go

```diff
@@ -229,18 +176,6 @@ func SKY_cipher_GenerateKeyPair(_arg0 *C.cipher__PubKey, _arg1 *C.cipher__SecKey
 	return
 }
 
-//export SKY_cipher_GenerateDeterministicKeyPair
```

Comment:
@stdevHan why removed ?

Reply:
Because move to https://github.com/simelo/libskycoin/blob/27e9c743fae2b479ddfeb5ed6dac252e3ddc31dd/lib/cgo/cipher.crypto.hw.go#L63